src/Microsoft.TemplateEngine.Cli/Alias/AliasSupport.cs

```diff
@@ -1,204 +1,203 @@
-                case AliasManipulationStatus.Removed:
-                    Reporter.Output.WriteLine(string.Format(LocalizableStrings.AliasRemoved, result.AliasName, string.Join(" ", result.AliasTokens)));
-                    return NewCommandStatus.Success;
-                case AliasManipulationStatus.RemoveNonExistentFailed:
-                    Reporter.Output.WriteLine(string.Format(LocalizableStrings.AliasRemoveNonExistentFailed, result.AliasName));
-                    return NewCommandStatus.AliasFailed;
-                case AliasManipulationStatus.Updated:
-                    Reporter.Output.WriteLine(string.Format(LocalizableStrings.AliasUpdated, result.AliasName, string.Join(" ", result.AliasTokens)));
-                    return NewCommandStatus.Success;
-                case AliasManipulationStatus.WouldCreateCycle:
-                    Reporter.Output.WriteLine(LocalizableStrings.AliasCycleError);
-                    return NewCommandStatus.AliasFailed;
-                case AliasManipulationStatus.InvalidInput:
-                    Reporter.Output.WriteLine(LocalizableStrings.AliasNotCreatedInvalidInput);
-                    return NewCommandStatus.InvalidParamValues;
-                default:
-                    return NewCommandStatus.OperationNotSpecified;
-            }
-        }
-
-        private static IReadOnlyList<string> FilterForAliasTokens(IReadOnlyList<string> inputTokens)
-        {
-            List<string> aliasTokens = new List<string>();
-            bool nextIsAliasName = false;
-            bool skipNextToken = false;
-            string? aliasName = null;
-
-            foreach (string token in inputTokens)
-            {
-                if (nextIsAliasName)
-                {
-                    aliasName = token;
-                    nextIsAliasName = false;
-                }
-                else if (skipNextToken)
-                {
-                    skipNextToken = false;
-                    continue;
-                }
-                else if (string.Equals(token, "-a", StringComparison.Ordinal) || string.Equals(token, "--alias", StringComparison.Ordinal))
-                {
-                    if (!string.IsNullOrEmpty(aliasName))
-                    {
-                        // found multiple alias names, which is invalid.
-                        aliasTokens.Clear();
-                        aliasName = null;
-                        return aliasTokens;
-                    }
-
-                    nextIsAliasName = true;
-                }
-                else if (token.Equals("--debug:custom-hive", StringComparison.Ordinal))
-                {
-                    skipNextToken = true;
-                }
-                else if (!token.StartsWith("--debug:", StringComparison.Ordinal))
-                {
-                    aliasTokens.Add(token);
-                }
-            }
-
-            return aliasTokens;
-        }
-    }
-}
+//#nullable enable
```

Comment:
I would prefer if we just removed whole file, and look at history when we want to re-introduce functionality.
Just my 2 cents...